### DISCUSSION TITLE
Getting an external script (CLI) running in Astro environment?

### DISCUSSION THREAD

**Discussion by czottmann** [withastro/docs#1436] [Ideas]
I was wondering about the best way to have an external script (for CLI use) run in the context of the app, so the environment is set up and all. I've checked the documentation but couldn't find anything — maybe I've overlooked it…

Thanks!

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

This no longer seems to be a viable discussion since a lot happened since 2022...

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!
